FFmpegInterop/FFmpegInteropMSS.h

```diff
@@ -24,7 +24,6 @@
 #include "MediaThumbnailData.h"
 
 using namespace Platform;
-using namespace Windows::Foundation;

```

Comment:
Why are we removing this, was there some naming collision?

Reply:
If it's just a stylistic change, I don't mind, I'm approving anyway

Reply:
There were compilation errors from ambiguous symbols after I included wrl.h. In general it's bad practice to write "using namespace" like this in header files. [C++ Core Guidelines SF.7: Don’t write using namespace at global scope in a header file](https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#sf7-dont-write-using-namespace-at-global-scope-in-a-header-file)